src/configuration/FirestoreConfigurationContext.tsx

```diff
@@ -43,8 +43,16 @@ import {useResubscribeToggle} from '@atb/utils/use-resubscribe-toggle';
 
 export const defaultVatPercent: number = 12;
 
+export type AppTextKey =
+  | 'travelAidSubText'
+  | 'travelAidHowToTitle'
+  | 'travelAidHowToContent'
+  | 'travelAidKeepInMindTitle'
+  | 'travelAidKeepInMindContent'
```

Comment:
I would advice against having all of these in Firestore. Lets talk in the office tomorrow, also with designer.

Reply:
The project owner needs to be involved in decisions like removing this from the remote config, it was decided on meetings.

Reply:
I created a Slack thread for this question:
https://mittatb.slack.com/archives/C02EEG7D8EL/p1729151816404169